packages/preview/hhn-unitylab-thesis-template/0.0.2/template/90-Document/90-Document.typ

```diff
@@ -0,0 +1,28 @@
+// General Configuration Template
+#import "../../src/config.typ": *
```

Comment:
You can't access this file from the template. If you expect the user to edit it, it should be in the template directory too, otherwise you should re-export its content and import it with the `@preview/...` import.